primerpedia.js

```diff
@@ -19,11 +19,17 @@
  */
 
 var wikipediaUrl = "https://en.wikipedia.org";
-var apiUrl = wikipediaUrl + "/w/api.php?";
-var apiExtractsQuery = "action=query&prop=extracts&exintro&indexpageids=true&format=json";
 var requestTimeoutInMs = 3000;
 var requestCallbackName = "requestCallback";
 
+var apiUrl = wikipediaUrl + "/w/api.php?";
+
+var apiExtractsQuery = "action=query&prop=extracts&exintro&indexpageids=true&format=json";
+var searchResultQuery = "?action=edit&amp;section=0";
```

Comment:
This should probably be named `editIntroUrlSuffix` or something like that.

Reply:
You marked 3 lines of variable assignments - which one did you mean?

Reply:
I meant the one immediately above the comment: the variable is named `searchResultQuery` but its URL clearly indicates its purpose does not match that name. I assume this was an overlook.

Reply:
Ah, ok. Well, this is the Query Part of the URL, hence the Name including Query but your Name seems fine.

Reply:
Sure, we can leave the "Query" part if it makes more sense: `editIntroQuery`.